src/System.IO.Compression/tests/ZipArchive/zip_CreateTests.cs

```diff
@@ -149,10 +149,58 @@ public static void CreateUncompressedArchive()
             }
         }
 
+        [Fact]
+        public static void CreateNormal_VerifyDataDescriptor()
+        {
+            using var memoryStream = new MemoryStream();
+            // We need an non-seekable stream so the data descriptor bit is turned on when saving
+            var wrappedStream = new WrappedStream(memoryStream, true, true, false, null);
```

Comment:
nit, not worth going back and changing now, but when passing booleans into a method, it's nice to name them since it often makes it clearer what's going on. eg `Foo(bar, baz, enableFrob:true)`

Reply:
sometimes also nulls.